tests/api/runtime_handlers/test_daskjob.py

```diff
@@ -33,13 +34,13 @@ def custom_setup(self):
         # initializing them here to save space in tests
         scheduler_pod_labels = {
             "app": "dask",
```

Comment:
what about the `"app"` label and `"user"` label?

Reply:
&